lms/djangoapps/courseware/access.py

```diff
@@ -304,6 +358,51 @@ def can_view_courseware_with_prerequisites():  # pylint: disable=invalid-name
     return _dispatch(checkers, action, user, course)
 
 
+COURSE_OVERVIEW_SUPPORTED_ACTIONS = [  # pylint: disable=invalid-name
+    'load',
+    'load_mobile',
+    'view_courseware_with_prerequisites'
```

Comment:
Nit: add a trailing comma `,` on the last entry so future additions have clean diffs.